.isort.cfg

```diff
@@ -0,0 +1,6 @@
+[settings]
+force_alphabetical_sort = True
```

Comment:
Currently an empty line is added between "from" and "import":
```diff
 from plone.uuid.interfaces import IUUID
+
 import json
```

We did not do that until now. I therefore prefer to not have the empty line.

**Problem:**
This is set by `lines_between_types=1`, which is set as [side effect of `force_alphabetical_sort = True`](https://github.com/timothycrosley/isort/blob/d756957bad7bb3b5acaf3a2842c999f00b2fd83e/isort/isort.py#L76-L80).

**Solution:**
Do not use
```
force_alphabetical_sort = True
```
but write it out and change `lines_between_types` to `0`:
```
force_alphabetical_sort_within_sections = True
no_sections = True
lines_between_types = 0
from_first = True
```

@Rotonen what's your position on that?